importer/src/main/java/org/hiero/mirror/importer/parser/record/transactionhandler/CryptoTransferTransactionHandler.java

```diff
@@ -2,12 +2,89 @@
 
 package org.hiero.mirror.importer.parser.record.transactionhandler;
 
+import com.hederahashgraph.api.proto.java.AccountAmount;
+import com.hederahashgraph.api.proto.java.NftTransfer;
+import com.hederahashgraph.api.proto.java.TokenTransferList;
 import jakarta.inject.Named;
+import java.util.List;
+import org.hiero.mirror.common.domain.entity.EntityId;
+import org.hiero.mirror.common.domain.transaction.HookExecutionCollector;
+import org.hiero.mirror.common.domain.transaction.RecordItem;
+import org.hiero.mirror.common.domain.transaction.Transaction;
 import org.hiero.mirror.common.domain.transaction.TransactionType;
 
 @Named
 class CryptoTransferTransactionHandler extends AbstractTransactionHandler {
 
+    @Override
+    protected void doUpdateTransaction(Transaction transaction, RecordItem recordItem) {
+        final var transactionBody = recordItem.getTransactionBody().getCryptoTransfer();
+        HookExecutionCollector hookExecutionCollector = HookExecutionCollector.create();
```

Comment:
```suggestion
        final var hookExecutionCollector = HookExecutionCollector.create();
```

and a lot other places can also use `final var`